src/2_add_cots_priority.jl

```diff
@@ -3,24 +3,26 @@ include("common.jl")
 # Load required data.
 canonical_file = find_latest_file(OUTPUT_DIR)
 RRAP_lookup = GDF.read(canonical_file)
-cots_priority = CSV.read(joinpath(DATA_DIR, "TargetReefs_202425.csv"), DataFrame)
+cots_target = CSV.read(joinpath(DATA_DIR, "COTS_Target_Reefs_Cull_Data_2024_25.csv"), DataFrame)
```

Comment:
No action needed.

An improvement could be to have a settings/configuration file specifying which files are inputs. That way we only need to change the config rather than messing with code each time the dataset changes.